Woof/Woof/View/SitterFlow/ScheduleView/ScheduleView.swift

```diff
@@ -7,13 +7,21 @@ struct ScheduleView: View {
             Text(scheduleTitle)
                 .navigationTitle(navigationTitle)
                 .navigationBarTitleDisplayMode(.inline)
+                .toolbar {
+                    ToolbarItem(placement: .navigationBarTrailing) {
+                        NavigationLink(navigationLinkTitle) {
+                            CreateNewWalkingView()
+                        }
+                    }
+                }
         }
     }
 
     // MARK: - Private interface
 
     private let scheduleTitle = "Walking schedule"
     private let navigationTitle = "My Schedule"
+    private let navigationLinkTitle = "add walking"
```

Comment:
```suggestion
    private let navigationLinkTitle = "Add walking"
```

or replace with "+" sign